### DISCUSSION TITLE
Updating an episode to make a better starting point

### DISCUSSION THREAD

**Discussion by brownsarahm** [carpentries/trainers#171] [Show and tell]
a [PR was submitted to mark an episode as avoid for demos](https://github.com/carpentries/instructor-training/pull/1346) because it had a very small dependency on the previous episode, instead of accepting that, I [proposed](https://github.com/swcarpentry/python-novice-inflammation/pull/986/files) to the lesson maintainers that we add the code (2 lines) to the episode to remove the dependency. 

The maintainer recommended using a `.prereq` box with an explanation of when those lines are/not needed.  

Sharing here in case it's helpful for others. 

```
> ## Episode Prerequisites
>
> If you are continuing in the same notebook from the previous episode, you already
> have a `data` variable and have imported `numpy`.  If you are starting a new 
> notebook at this point, you need the following two lines:
>
> ~~~
> import numpy
> data = numpy.loadtxt(fname='inflammation-01.csv', delimiter=',')
> ~~~
> {: .language-python}
{: .prereq}
```

**Comment by brownsarahm** [comment]
@carpentries/core-team-curriculum this might be relevant as you all will be helping with starting point maintenance more generally across all lessons.

**Comment by ErinBecker** [comment]
Thank you @brownsarahm. The Curriculum Team has reviewed this conversation and agrees that this is a good fix!
